includes/class-wc-subscription-data-copier.php

```diff
@@ -0,0 +1,463 @@
+<?php
+/**
+ * Woocommerce Subscriptions Data Copier
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+class WC_Subscription_Data_Copier {
```

Comment:
This was a typo. I've changed it to `WC_Subscriptions_Data_Copier`.

On the `WCS` vs `WC_Subscriptions` thing I think we should use `'WC_Subscriptions'` where ever we can because we don't have a monopoly on that acronym given wc shipping and wc services also use WCS. Where I do think it makes sense to use WCS is when the class, filter, or function name also includes the word subscription. eg `wcs_get_subscription()`, or when the class name would be excessively long if we were to spell it out, or it's reasonable to assume a third party wouldn't define the same class. In this case, copying data doesn't seem unique enough to subscriptions to allow the use of WCS imo.

Reply:
🙌  I like this discussion of our prefixes/naming conventions. Might be something we could document in our handbook or in repo markdown docs.